### PR TITLE
Quality control for human biomolecular data page

### PR DESCRIPTION
**Pull request by lauportell**

### CODE REVIEW

File: quality-control/human-biomolecular-data.md
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:71]:
```suggestion
- {% tool "graf-pop" %}: GRAF-pop is a software tool that infers the subject ancestry. For the estimation it uses about 100,000 SNPs from genotype datasets, which can be in PLINK and VCF format. 
```

File: quality-control/human-biomolecular-data.md
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:72]:
```suggestion
- {% tool "graf-sex" %}: GRAF-sex determines subject sexes using the genotypes. It’s used to validate the self-reported sexes in phenotype datasets.
```

File: quality-control/human-biomolecular-data.md
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:80]:
```suggestion
- **Negative and positive controls**: including negative controls it’s important to detect contamination in the reagents, consumables, and general laboratory environment. Including negative controls in each run of diagnostic molecular testing based on RT-PCR, for example, will avoid false positive results resulting from cross-contamination. As well as positive controls for the target sequence/s used in the individual assays, to ensure no false negative results due to reagents failure. For example, the study of a housekeeping human gene would ensure quality of the sample collection procedure, as it controls that the sample has human material and the quality of the analytical procedure, making it possible to check that there is no PCR inhibition and the reagents are working properly.
```

File: quality-control/human-biomolecular-data.md
Comment (EvaGarciaAlvarez) [inline quality-control/human-biomolecular-data.md:94]:
```suggestion
Besides sample metadata, assay metadata describing how the results were generated is also necessary. This metadata will depend on the type of assay (antigenic, molecular, target genes of the assay, serological, etc) and it should include all the used variables on which the results depend on.
```

File: quality-control/human-biomolecular-data.md
Comment (lauportell) [reply quality-control/human-biomolecular-data.md:4]: oh sorry that I missed them. Can you add Clementina to the Contributors yaml? I don't know her details.

File: quality-control/human-biomolecular-data.md
Comment (lauportell) [reply quality-control/human-biomolecular-data.md]: done!

File: quality-control/human-biomolecular-data.md
Comment (lauportell) [reply quality-control/human-biomolecular-data.md]: Added! :)

**Comment by lauportell** [comment]
All comments are solved - I think this can be merged once it has a final review :)

**Review by bedroesb — APPROVED** [review]
Looks also good from a technical view!

**Comment by bedroesb** [comment]
@lauportell I noticed that there are some differences between the tool names in the name column vs when it is mentioned in its description. Could you quickly align them?

**Comment by bedroesb** [comment]
For example SAMtools vs Sam tools

**Pull request merged**
